File: Assignment_1/programAQ4.c
///////////////////////////////////////////////////////////
//
//  File name : Assigment1Question4.c
//  Descreption : Used to accpect one number & chech whether is divisible by 5 or not
//  Author : Kartik Ganesh Jare
//  Date : 09/05/25
//  
///////////////////////////////////////////////////////////

#include<stdio.h>

typedef int BOOL;
#define TRUE 1
#define FALSE -1

BOOL Check(int iNo)
{
    if((iNo %5)== 0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number:");
    scanf("%d",&iValue);

    bRet = Check(iValue);

    if(bRet == TRUE)
    {
        printf("Divisble by 5\n");
    }
    else
    {
        printf("Not Divisble by 5\n");
    }

    return 0;
}

File: Java_Assignment/Assignment_1/programAQ4.java
///////////////////////////////////////////////////////////
//
//  File name : Assigment1Question4.java
//  Descreption : Used to accpect one number & chech whether is divisible by 5 or not
//  Author : Kartik Ganesh Jare
//  Date : 31/05/25
//  
///////////////////////////////////////////////////////////
import java.io.*;

class Check
{
    public Boolean CheckDivisible(int iNo)
    {
        if((iNo % 5) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class programAQ4
{
    public static void main(String A[]) throws Exception
    {
        int iValue = 0;
        Boolean bRet = false;

        BufferedReader bobj = new BufferedReader(new InputStreamReader(System.in)); 

        System.out.println("Enter number: ");
        iValue = Integer.parseInt(bobj.readLine());

        Check cobj = new Check();
        bRet = cobj.CheckDivisible(iValue);

        if(bRet == true)
        {
            System.out.println("Divisble by 5");
        }
        else
        {
            System.out.println("Not Divisble by 5");
        }
    }
}